facets_dive/components/facets_dive_controls/facets-dive-controls.html

```diff
@@ -216,7 +216,7 @@
               class="dropdown-content"
```

Comment:
mahima's mocks hide these controls but we agreed also to change "Position" to "Scatter" for both axes for those dropdowns as well.

Reply:
Renamed in 955e753